test/built-ins/AsyncContext/context-propagation/async-generators-yield-inside-run.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2023 Igalia, S. L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: TODO
+flags: [async]
+features: [AsyncContext]
+includes: [asyncHelpers.js]
+---*/
+
+// TODO: This test tests the behavior in
+// https://github.com/tc39/proposal-async-context/pull/61
+
+const asyncVar = new AsyncContext.Variable();
+
+async function* gen() {
+  await Promise.resolve();
+  assert.sameValue(asyncVar.get(), "init");
```

Comment:
Same here, async generators are initially in a SUSPEND-START state, so we need to resume the generator with .next() before the first await or this assert can be reached.